test/testcafe/spec/ReEnrollAuthenticatorWarningPasswordView_spec.js

```diff
@@ -130,14 +131,14 @@ test
     });
   });
 
-test
+test.meta('v3', false)
```

Comment:
I had to disable this test because it expects the previous transaction's elements to stay in the UI even though they are not included with this error message (it is a terminal transaction). 

I checked master and this test has not been changed. Should I make the changes in v3 to reserve that step's data to re-render on this transaction? @shawyu-okta

Reply:
Let's file a JIRA for this and mark it here. We can followup with the change if needed separately